docs/_util.js

```diff
@@ -0,0 +1,28 @@
+function stripIndent (rawString) {
+    const match = rawString.match(/^[ \t]*(?=\S)/gm);
+
+    if (!match) {
+        return rawString;
+    }
+
+    const minIndent = Math.min.apply(Math, match.map(x => x.length));
+
+    const reMinIndent = new RegExp(`^[ \\t]{${minIndent}}`, 'gm');
```

Comment:
Why do you need to check for either a whitespace, a `\`, or a tab (or, really, why are you checking for a space or tab at the beginning again)? Were you coming across a bug?

Reply:
This function was pulled from https://github.com/rackerlabs/helix-ui/blob/master/docs/_nunjucks/code.tag.js#L16-L27 with slight modification, and this regexp comes from https://github.com/rackerlabs/helix-ui/blob/master/docs/_nunjucks/code.tag.js#L24.

Essentially, what the function is doing is normalizing indentation so that we don't have something like this...
![screen shot 2018-03-09 at 1 42 25 pm](https://user-images.githubusercontent.com/545605/37226485-b7b25804-239f-11e8-8b28-f25d556e9c5a.png)

Reply:
Ah, kk. Makes sense.